psydac/core/kernels.py

```diff
@@ -1,7 +1,12 @@
 import numpy as np
 
-
-def eval_fields_3d_no_weights(nc1: int, nc2: int, nc3: int, pad1: int, pad2: int, pad3: int, f_p1: int, f_p2: int,
+# =============================================================================
+# Field evaluation functions 
```

Comment:
General comment: throughout Psydac we always use the symbols `x, y, z` to identify the physical coordinates, while we use `x1, x2, x3` (sometimes `xi1, xi2, xi3` or `eta1, eta2, eta3`) to identify the logical coordinates.

Most functions in this file take arguments that refer to the logical coordinates, not the physical ones, hence the docstrings should be changed accordingly.

Reply:
Docstrings changed in [b8b1cea](https://github.com/pyccel/psydac/pull/214/commits/b8b1cea2241f1cb78baccab3fd96192b277d57f5)